### ISSUE TITLE
 wasmtime-X.0.0-py3-none-any.whl doesn't include linux-x86_64

### ISSUE THREAD

**Issue by JulienG515** [bytecodealliance/wasmtime-py#207]
Hi,

The version py3-none-any.whl doesn't include build for linux-x86_64.

Thank you,
BR,

Julien

**Comment by alexcrichton** [comment]
Of the [files published](https://pypi.org/project/wasmtime/#files) the intention is that wasmtime-17.0.0-py3-none-manylinux1_x86_64.whl is used by x86_64 linux. The none-any is there for MinGW Windows IIRC, although I also sort of forget.

I'm also no expert on Python packaging, however, so do you know why the tooling you're using ended up using the none-any whl? Or do you have another use case for this wheel?

**Comment by whitequark** [comment]
> The none-any is there for MinGW Windows IIRC, although I also sort of forget.

Yeah, there's no way to separately distribute packages for MinGW, and this is the workaround.
